doc/contributions-guide.md

```diff
@@ -115,3 +107,7 @@ are welcome, please use the [netopenconfig Google
 group](mailto:netopenconfig@googlegroups.com), or the [public repository
 issues](https://github.com/openconfig/public/issues). 
 
+[0]: Previously, changes were merged in an upstream repository, but since
+2022-05-01 `openconfig/public` is the source of truth for OpenConfig models.
+Please note that this requires stricter consideration of version numbers
+that were previously changed upstream.
```

Comment:
I am ok with this footnote, however, do we need the historical context here?  The git history and PR history will indirectly relate this information if people are very interested?

Reply:
Made the edit to remove it.